.eslintrc.js

```diff
@@ -224,6 +224,13 @@ module.exports = {
 				],
 			},
 		},
+		{
+			files: [ 'packages/components/src/**/*.js' ],
```

Comment:
Let's include more extensions (js, ts, tsx, jsx)

```suggestion
			files: [ 'packages/components/src/**/*.[tj]s?(x)' ]
```

Reply:
Sounds good, I'll make sure the list above gets updated based on this change 👍